packages/vuetify/src/labs/VDatePicker/VDatePicker.tsx

```diff
@@ -175,9 +200,10 @@ export const VDatePicker = genericComponent<VDatePickerSlots>()({
             ) : (
               <div class="v-date-picker__input">
                 <VTextField
-                  v-model={ inputModel.value }
+                  modelValue={ inputModel.value[0] }
+                  onUpdate:modelValue={ v => updateFromInput(v, 0) }
```

Comment:
updateFromInput is only used here. Does it need to accept an index arg?

Reply:
It will be needed for range picker as well, at which point it'll be moved somewhere else.